public/locales/pt-BR/translation.json

```diff
@@ -1,4 +1,8 @@
 {
   "dashboard": "Painel",
-  "agreements": "Acordos"
+  "agreements": "Acordos",
+  "validation": {
+    "required": "{{field}} é obrigatório"
+  },
+  "title": "Título"
```

Comment:
Only the title is being translated, you need to translate the other fields as well

Reply:
I noticed that you added a logic that should work for the other fields automatically, but only the title is being translated.

Reply:
Yes, we have to translate all fields! This was just an example so you can get from here